### ISSUE TITLE
Add informative __repr__ methods

### ISSUE THREAD

**Issue by kgbaird** [smart-on-fhir/client-py#15]
This library would be much more user-friendly if the custom objects included informative `__repr__` methods on them.

**Comment by p2** [comment]
Yes sure! What would you like to see in there? I often use `as_json()` but am guessing it's overkill to add that to the repr.

**Comment by kgbaird** [comment]
In our use, it appears most of the objects are just key-values of attributes. Having a visual of the attributes would be nice. `as_json()` doesn't sound that bad to me, but it probably is a bit much.
